packages/platform/http/ttl.js

```diff
@@ -31,10 +31,10 @@ export function createTtlHandler() {
 				if (cachedResponse) {
 					// If a previously cached response has not yet expired, we can serve
 					// this without performing an http request
-					if (cachedResponse.expires > now + ttl) {
+					if (cachedResponse.expires > now) {
             options.context.servedFromCache = true;
```

Comment:
Can you do me a favour and format this fill again, as it has a mixture of spaces and tabs? Thank you.

Reply:
Done